openerp/addons/test_new_api/tests/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import test_onchange
 from . import test_field_conversions
 from . import test_attributes
+from . import test_cache
```

Comment:
Please rename this file. This is not only about cache issues, this really is specific issues about one2many fields.